drivers/SmartThings/matter-sensor/profiles/motion-illuminance-battery.yml

```diff
@@ -0,0 +1,16 @@
+name: motion-illuminance-battery
```

Comment:
This profile is a copy of `matter-motion-battery-illuminance` but I duplicated it so that it would follow the naming scheme that is used in `do_configure` for updating the profile

Reply:
Is the plan to make copies of all the existing profiles that don't follow the naming scheme, and then add code to switch from the old profile to the new one, and finally we remove the old profiles from the driver once we know that no devices are still using them?

Reply:
This is what I was imagining we would do. At the moment, only two profiles don't fit the naming scheme: `matter-motion-battery-illuminance`, and `motion-battery` which has an incorrect `name:` field despite the filename being correct. The plan would be to make a duplicates of both and remove the old profiles once they are no longer used, just like you said.